private_repo/ansible/inventory/dev/KnowledgePlatform/secrets.yml

```diff
@@ -47,5 +47,26 @@ lp_vault_youtube_api_key:                       # youtube api token if you want
 lp_vault_graph_passport_key: "long-secret-to-calm-entropy-gods"
 
 # update if seperate object storage is used
+# If cloud_service_provider is aws then update secret access key
+# Example: cloud_public_storage_secret: "xkxjfjsdsfjdfjdhgjfhgjdhfgjdgjdfgdjgjdgdjhgjhd"
+
+# If cloud_service_provider is gcp then update with private-key which is in service account json file
+# Example: cloud_public_storage_secret: "-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n"
+
+# If cloud_service_provider is azure then update with storage account key
+# Example: cloud_public_storage_secret: "BfoOQwA6UFI7URwkFjkjdjsdhjhjhjsdhjfM//lsTbXDmHOFR5GqEwAwzyrKXOUDywojCpOC/g=="
 cloud_private_storage_secret: "{{ cloud_public_storage_secret }}"
-cloud_artifact_storage_secret: "{{ cloud_public_storage_secret }}"
\ No newline at end of file
+cloud_artifact_storage_secret: "{{ cloud_public_storage_secret }}"
+
+### Lern BB
+core_vault_sunbird_fcm_account_key: "" # Firebase Cloud Messaging API Key
+sunbird_msg_91_auth: ""              # API key for sending OTP SMS
+sunbird_mail_server_password: ""   # Email server password
+dp_vault_data_exhaust_token: ""    # slack api token
+core_vault_sunbird_api_auth_token: # copy value form variable core_vault_sunbird_api_auth_token from core/secrets.yml
+dp_vault_pgdb_password: "change.postgres.password"         # postgres password for analytics db
```

Comment:
These vars cannot be kept under `# Sensible defaults` section as the adopter needs to fill these. Please move them to mandatory section.

Reply:
@keshavprasadms  moved the vars to the respective sections. Please review now